std_msgs/Header header

##string user_id
##float64 direction
##float64 velocity

string user_id
float64 direction
float64 velocity
float64 acc
float64 threshold_upper
float64 threshold_low
float64 avg
float64 signal
int64 state
int64 LCSS_state

##int64 Imu_state
##int64 Lidar_state
